CHANGES.rst

```diff
@@ -52,15 +52,15 @@ Bugs fixed
 * An unnecessary slow-down at import time was removed from ``Cython.Distutils``.
   Original patch by Anthony Sottile.  (Github issue :issue:`4224`)
 
-* Python modules were not automatically recompiled when only their ``.pxd`` file changed.
+* Pure Python modules were not automatically recompiled when only their ``.pxd`` file changed.
   Patch by Golden Rockefeller.  (Github issue :issue:`1428`)
 
 * The signature of ``PyFloat_FromString()`` in ``cpython.float`` was changed
   to match the signature in Py3.  It still has an automatic fallback for Py2.
   (Github issue :issue:`3909`)
 
 * A compile error on MSVC was resolved.
-  Patch by David Woods.  (Github issue :issue:`4202`)
+  Patch by David Woods.  (Github issue #4202)
```

Comment:
Please undo this change.

Reply:
oh yeah sorry

Reply:
[resolved](https://github.com/cython/cython/pull/4241/commits/365bc99d260d0edd546566eecedaa90b316bdbc0)